helm-charts/falcon-sensor/values.yaml

```diff
@@ -19,9 +19,9 @@ daemonset:
   #   necessarily unless running within environments such as OpenShift.
   securityContext:
     runAsUser: 0
-    privileged: false
-    readOnlyRootFilesystem: true
-    allowPrivilegeEscalation: false
+    privileged: true
+    readOnlyRootFilesystem: false
+    allowPrivilegeEscalation: true
```

Comment:
Why the updated privileges and readOnly to false?

Reply:
@shawndwells does the Linux kernel sensor require a r/w root filesystem, or no?

Reply:
@shawndwells the updated privileges are required for the sensor to work correctly on the RHCOS nodes. Otherwise, deployment fails on OpenShift.